examples/celix-examples/dm_example_cxx/CMakeLists.txt

```diff
@@ -22,9 +22,10 @@ add_subdirectory(phase2)
 add_subdirectory(phase3)
 add_subdirectory(phase3_locking)
 
-add_celix_container("dm_example_cxx"
+add_celix_container(dm_example_cxx
+    FAT
     GROUP cxx_examples
-    BUNDLES
+    EMBEDDED_BUNDLES
```

Comment:
Changing a test instead of adding one. Intended?

Reply:
Yes, because there are enough examples using BUNDLES, IMO updating 1 to EMBEDDED_BUNDLES is enough and this does not increase the nr of celix containers to build.